tests/e2e/config.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+const site = process.env.CMK_SITE || 'cmk';
+
+const cfg = {
+  //Checkmk
+  site: site,
+  cmkUser: process.env.CMK_ADMIN || '',
+  cmkPassword: process.env.CMK_PASSWORD || '',
+  automationUser: process.env.CMK_AUTOMATION || '',
+  automationPassword: process.env.CMK_PASSWORD || '',
+
+  // Playwright
+  playwrightToCheckmkUrl: process.env.PLAYWRIGHT_TO_CHECKMK_URL || '',
```

Comment:
maybe the `|| ''` ist a bit dangerous? i would have preferred that if fails loudly if the environment variables are not defined. should make finding the problem easier.